charts/gardener-dashboard/charts/runtime/templates/configmap.yaml

```diff
@@ -0,0 +1,341 @@
+# SPDX-FileCopyrightText: 2021 SAP SE or an SAP affiliate company and Gardener contributors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: gardener-dashboard-configmap
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: gardener-dashboard
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+data:
```

Comment:
We could wrap it with 
```
{{- with .Values.global }}
...
{{- end }}
```
to not have this very long pathnames

Reply:
we agreed that, for better search-ability of the properties, we do not simplify here even though it would improve the readability